src/TrafficSign/TrafficSign.types.ts

```diff
@@ -0,0 +1,126 @@
+/**
+ * Properties of the TrafficSign component
+ */
+
+export type TrafficSignProps = {
+  /**
+   * The type of the traffic sign
+   */
+  type:
+    | "barrier"
+    | "barrier2S"
+    | "beaconArrowL"
+    | "beaconArrowR"
+    | "beaconHatchedWide"
+    | "beaconHatchedWideL"
+    | "beaconHatchedWideR"
+    | "beaconL"
+    | "beaconL2S"
+    | "beaconR2S"
+    | "beaconR"
+    | "caution"
+    | "children"
+    | "crossroads"
+    | "curveL"
+    | "curveR"
+    | "cyclistsCrossing"
+    | "cyclistsCrossingL"
+    | "cyclistsCrossingR"
+    | "deadEnd"
+    | "directionBoardL"
+    | "directionBoardR"
+    | "directionBoardWideL"
+    | "directionBoardWideR"
+    | "diversionL"
+    | "diversionR"
+    | "endOfLimitations"
+    | "exitHighway"
+    | "exitInnerCityRoad"
+    | "exitLimitedAccessRoad"
+    | "fileUnknown"
+    | "giveWay"
+    | "highway"
+    | "highwayEnd"
+    | "icy"
+    | "limitedAccessRoad"
+    | "limitedAccessRoadEnd"
+    | "livingStreet"
+    | "livingStreetEnd"
+    | "minimumSpeed"
+    | "minimumSpeedEnd"
+    | "narrowRoad"
+    | "narrowRoadL"
+    | "narrowRoadR"
+    | "noEntry"
+    | "noOvertaking"
+    | "noOvertakingEnd"
+    | "noOvertakingTrucks"
+    | "noOvertakingTrucksEnd"
+    | "noTraffic"
+    | "oneWayLeft"
+    | "oneWayRight"
+    | "oneWayStraight"
+    | "parkingGarage"
+    | "parkingLot"
+    | "passL"
+    | "passR"
+    | "pedestrianCrossing"
+    | "pedestrians"
+    | "priorityRoad"
+    | "priorityRoadEnd"
+    | "rightOfWay"
+    | "roadWorks"
+    | "roundabout"
+    | "sCurveL"
+    | "sCurveR"
+    | "slipperyRoad"
+    | "speedLimit"
+    | "speedLimitEnd"
+    | "speedLimitZone"
+    | "speedLimitZoneEnd"
+    | "steepAscent"
+    | "steepDescent"
+    | "stop"
+    | "straightOrLeft"
+    | "straightOrRight"
+    | "trafficLight"
+    | "trafficQueues"
+    | "tunnel"
+    | "turnAheadL"
+    | "turnAheadR"
+    | "turnL"
+    | "turnR"
+    | "unevenRoad"
+    | "urbanArea"
+    | "urbanAreaEnd"
+    | "warningPedestrianCrossing"
+    | "weightLimit"
+    | "widthLimit";
+  /**
+   * The coordinates of the sign in Fr0 [x,y]
+   */
+  points: number[];
+  /**
+   * The value of the sign (e.g., speed limit)
+   */
+  value?: string;
+  /**
+   * A rotation angle in degrees
+   */
+  angle?: number;
+  /**
+   * The scale of the image
+   */
+  scale?: number;
```

Comment:
What unit is this in? I entered a value of -100 and something rendered but I can't quite work out what the scale is actually doing.

<img width="1420" alt="image" src="https://github.com/IPG-Automotive-UK/react-ui/assets/8976377/234151cf-2745-48bb-93b4-39dd21ca6a6e">

Reply:
Is negative somehow flipping as well? I assume this is allowed/expected syntax?

Reply:
Yeah, so this is a factor that scales the size of image/traffic sign. The native behavior of the library is that a negative value will 'flip' the image. I can of course disable this and only allow positive numbers, but my logic was to keep it as close to the native thing as possible. For now I have added a comment in the types, hopefully explaining it better.

![image](https://github.com/IPG-Automotive-UK/react-ui/assets/67595415/68fd67b9-2a61-4568-8e68-7f582ec0490d)